### PR TITLE
Made it easier to find esc files 3-17-18

### PR DESCRIPTION
**Pull request by lavaduderDev**
This adds to the item and base_scene gdscripts, a filter. There will not be other file types shown.

### CODE REVIEW

File: device/globals/item.gd [hunk 1]
```diff
@@ -4,7 +4,7 @@ extends "res://globals/interactive.gd"
 
 export var tooltip = ""
 export var action = ""
-export(String, FILE) var events_path = ""
+export(String, FILE,".esc") var events_path = ""
```
Comment (fleskesvor) [inline device/globals/item.gd]: Could you add a space after the second comma?

File: device/globals/scene_base.gd [hunk 2]
```diff
@@ -1,6 +1,6 @@
 extends Node
 
-export(String, FILE) var events_path = ""
+export(String, FILE,".esc") var events_path = ""
```
Comment (fleskesvor) [inline device/globals/scene_base.gd]: Here too.

**Comment by fleskesvor** [comment]
This is great. 🙂 Did you change anything in `device/globals/scene_main.tscn` or did Godot make those changes? I get the `index` and `events_path` changes too, so I'm guessing Godot adds some default values to .tscn files now.

**Comment by lavaduderDev** [comment]
Godot made those changes. I just figured I'd add it to the commit.

**Comment by fleskesvor** [comment]
Yeah, that's good then.

Now we'll just squash these three commits into one, and the PR is good to be merged. If you look at `git log`, you'll see that the last commit hash before your first one is `60ed5bad7f9da3d9f7b59ccdbad2408c12b9a9be`. Using that, we'll do what's called a rebase with the command

    git rebase -i 60ed5bad7f9da3d9f7b59ccdbad2408c12b9a9be

Which will put you in interactive mode with the following lines

    pick be448da Made it easier to find esc files 3-17-18
    pick 1232317 added space to item.gd file finder
    pick 723bc0e added space to scene_base.gd file finder

plus a brief overview of what your options are. Now edit the second and third line so that they look like

    pick be448da Made it easier to find esc files 3-17-18
    s 1232317 added space to item.gd file finder
    s 723bc0e added space to scene_base.gd file finder

and save the file. If you're using vim, thats `Esc`, `:` (colon) `w`, `q`, `Enter`, in that order.

Now, you'll see

    # This is a combination of 3 commits.
    # This is the 1st commit message:

    Made it easier to find esc files 3-17-18

    # This is the commit message #2:

    added space to item.gd file finder
    # This is the commit message #3:

    added space to scene_base.gd file finder

You can comment out the last two messages by adding a `#` (pound/hash mark) in front of them, and again save the file.

If you look at `git log` again, you'll see that your three commits have become one, and you can force push your branch to update this PR.

    git push --force origin filesimprove

And I'll be happy to merge your commit. 🙂

**Comment by lavaduderDev** [comment]
Uh... I did those two commits with github's built in editor.

**Comment by fleskesvor** [comment]
That's OK. Are you comfortable with command line Git though?

**Pull request merged**